$MirOS: ports/graphics/lcms/patches/patch-src_cmsmtrx_c,v 1.2 2005/04/28 20:24:19 tg Exp $
$OpenBSD: patch-src_cmsmtrx_c,v 1.1 2004/02/03 05:15:16 brad Exp $
--- src/cmsmtrx.c.orig	Fri Nov 25 11:10:54 2005
+++ src/cmsmtrx.c	Wed Jan 11 17:12:38 2006
@@ -797,9 +797,9 @@ void MAT3fromFix(LPMAT3 r, LPWMAT3 v)
 
 void VEC3scaleAndCut(LPWVEC3 r, LPVEC3 v, double d)
 {
-        r -> n[VX] = (int) floor(v -> n[VX] * d + .5);
-        r -> n[VY] = (int) floor(v -> n[VY] * d + .5);
-        r -> n[VZ] = (int) floor(v -> n[VZ] * d + .5);
+        r -> n[VX] = (icInt32Number) floor(v -> n[VX] * d + .5);
+        r -> n[VY] = (icInt32Number) floor(v -> n[VY] * d + .5);
+        r -> n[VZ] = (icInt32Number) floor(v -> n[VZ] * d + .5);
 }
 
 void MAT3scaleAndCut(LPWMAT3 r, LPMAT3 v, double d)
